packages/esm-patient-chart-app/src/index.ts

```diff
@@ -148,6 +148,14 @@ function setupOpenMRS() {
           moduleName,
         }),
       },
+      {
+        id: 'visit-detail-overview',
```

Comment:
```suggestion
        id: 'visit-detail',
```

?

Calling something "xyz-detail-overview" doesn't really make sense to me. Is it a detail view, or is it an overview? They're opposites.